keyboards/pistachio_pro/readme.md

```diff
@@ -0,0 +1,19 @@
+# PistachioPro
+
+![PistachioPro](https://ratelog.net/wp-content/uploads/2021/07/PistachioPro-scaled.jpg)
+
+This is a JIS layout TKL keyboard with 89 keys, encoder and OLED.
+
+* Keyboard Maintainer: [rate](https://github.com/7-rate)
+* Hardware Supported: PistachioPro PCB,
+* Hardware Availability: [Booth Shop](https://rates.booth.pm/items/2925303)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make pistachio_pro:default
+
+Flashing example for this keyboard:
+
+    make pistachio_pro:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

Comment:
Above this line, could you please add instructions on how to reset the keyboard into bootloader mode?

Reply:
Solved with 844d24